packages/forms/src/elements/tiles/Tiles.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  HTMLAttributes,
+  ChangeEventHandler,
+  useCallback,
+  useState,
+  ForwardRefExoticComponent,
+  PropsWithoutRef,
+  RefAttributes
+} from 'react';
+import PropTypes from 'prop-types';
+import { getControlledValue } from '@zendeskgarden/container-utilities';
+import { TilesContext } from '../../utils/useTilesContext';
+import { Tile } from './components/Tile';
+import { Description } from './components/Description';
+import { Icon } from './components/Icon';
+import { Label } from './components/Label';
+
+interface ITilesProps extends HTMLAttributes<HTMLDivElement> {
+  /** Value of the selected radio button */
+  value?: string;
+  /** Callback when a radio is selected */
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+  /** The name used to reference the value of the control. */
+  name: string;
+  /** Displays the tiles in their vertical arrangement */
+  isStacked?: boolean;
+}
+
+interface IStaticTilesExport<T, P>
+  extends ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>> {
+  Tile: typeof Tile;
+  Description: typeof Description;
+  Icon: typeof Icon;
+  Label: typeof Label;
+}
+
+/* eslint-disable react/display-name */
+/**
+ * Accepts all `<div>` attributes
```

Comment:
Without improved documentation here, it is very hard to understand how to use the `Tiles` component just by viewing the API. Unless there is a way to make Styleguidist pick up the `Tiles.Tile` etc. components below, there should probably be a minimal usage example along with showing how it should be incorporated with the `Grid`.

Reply:
I've added a new basic example and some documentation regarding layout and the static property components.